### ISSUE TITLE
cocoapod for osx requires missing crypto library

### ISSUE THREAD

**Issue by stanchiang** [MailCore/mailcore2#1766]
I installed pod 'mailcore2-osx' to my empty macOS project 
https://github.com/stanchiang/testmailcore

and got the following error
ld: library not found for -lcrypto
(screenshot linked)
https://www.dropbox.com/s/jwtpvjsnqsmwl7v/Screenshot%202018-07-15%2017.41.09.png?dl=0

my Mac version: 10.13.5, my xcode version: 9.4.1

I saw a previous closed issue where crypto was removed, not sure why my machine thinks it's needed https://github.com/MailCore/mailcore2/pull/1619

don't think it's something on my end since all I did was literally just instantiate a new Xcode Mac project, added your pod, and immediately got this error.

Any ideas?

**Comment by techistoner** [comment]
i have the same issue.
my pod file:
platform :osx, '10.11'

target 'target' do
    pod 'mailcore2-osx'
end

the error:
d: library not found for -lcrypto
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by jessetipton** [comment]
@stanchiang @techistoner Were you ever able to resolve this? Seeing the same issue with `platform :osx, '10.15'`.

**Comment by rgbworld** [comment]
Same issue here: `ld: library not found for -lcrypto`

macOS 10.15.2, Xcode 11.3.1
I installed MailCore 2 manually and set compiler settings per instructions. openssl has been replaced with libressl.

Other Linker Flags `-lctemplate -letpan -lxml2 -lsasl2 -liconv -ltidy -lz -lc++ -stdlib=libc++ -ObjC -lcrypto -lssl -lresolv -framework Security -framework IOKit`

Any suggestions on how to fix the compile error?

openssl version -a
LibreSSL 2.8.3
built on: date not available
platform: information not available
options:  bn(64,64) rc4(16x,int) des(idx,cisc,16,int) blowfish(idx) 
compiler: information not available
OPENSSLDIR: "/private/etc/ssl"
